docs/src/components/footer/EmailSubscribe.tsx

```diff
@@ -24,7 +24,13 @@ function searchParams(params: any) {
     .join('&');
 }
 
-export default function EmailSubscribe({ sx }: { sx?: SxProps<Theme> }) {
+export default function EmailSubscribe({
+  sx,
+  fullWidth,
+}: {
+  sx?: SxProps<Theme>;
+  fullWidth?: Boolean;
```

Comment:
Seems it's not used anywhere. I assume it comes from the first iteration (no problem to keep it)

Reply:
Good call, let me remove it!